OpenSim/Moco/MocoGoal/MocoContactImpulseTrackingGoal.h

```diff
@@ -234,29 +225,29 @@ namespace OpenSim {
             const IntegrandInput& input, double& integrand) const override;
         void calcGoalImpl(
             const GoalInput& input, SimTK::Vector& cost) const override {
-            cost[0] = input.integral / m_denominator;
+            cost[0] = (input.integral * input.integral) / m_denominator;
         }
         void printDescriptionImpl() const override;
 
     private:
         // PROPERTIES
-        OpenSim_DECLARE_LIST_PROPERTY(contact_groups, MocoImpulseTrackingGoalGroup,
+        OpenSim_DECLARE_LIST_PROPERTY(contact_groups, MocoContactImpulseTrackingGoalGroup,
             "Associate contact elements in the model with force data.");
         OpenSim_DECLARE_OPTIONAL_PROPERTY(external_loads, ExternalLoads,
             "Experimental contact force data.");
         OpenSim_DECLARE_PROPERTY(external_loads_file, std::string,
             "Experimental contact force data as an ExternalLoads XML file.");
         OpenSim_DECLARE_PROPERTY(impulse_axis, int,
-            "User specifies which axis to track the GRF impulse: "
-            "'1': X, '2': Y, '3': Z"
+            "User specifies which axis to track the ground reaction force impulse: "
```

Comment:
Again, this description should use language to describe what it is, rather than how a user will interact it with. For example: "The axis of the ground reaction force impulse component to be tracked in the goal."